### ISSUE TITLE
Name / Domain

### ISSUE THREAD

**Issue by kjoscha** [kjoscha/lichtenberg#7]
...wäre natürlich auch nicht schlecht. Vorschläge?

**Comment by andneuma** [comment]
Jo, das ist so ein bisschen die Frage, weil man denke ich verschiedene Richtungen einschlagen kann... mal so gebrainstormt:

1. Man macht das ganze als Seite für Lichtenberg, wo man unkommerzielle Strukturen abbildet, also zB Lichtenberg-unkommerziell.org oder Lichtenberg-fuer-alle.org / Lichtenberg4all.org

2. Man macht das für gesamt Berlin und bietet die Plattform anderen Leuten an. Dann wäre könnte das zB Berlin-unkommerziell.org / Berlin-fuer-alle.org / berlin4all.org sein

=> Eigentlich wär das glaube ich mein Favorit, sofern andere Gruppen da Interesse haben.

3. Man macht das speziell für refugees als Willkommen-in-XYZ-Seite. Wobei ich finde, dass man sich damit auch stark in die Nähe von so Willkommens-Initiativen stellt (Was natürlich nichts per se schlechtes sein muss). Wär dann vielleicht sowas wie willkommen-in-Lichtenberg.org? Könnte halt auch vom Rathaus sein, so eine Seite...

**Comment by andneuma** [comment]
Also, wo hosten wir das ganze jetzt? M ist glaub ich erstmal raus. Von mir aus können wir das gern dort machen, ich frag mal an, ob wir das über Aktiv in Lberg machen können.

**Comment by kjoscha** [comment]
>Von mir aus können wir das gern dort machen

was ist dort?

**Comment by kjoscha** [comment]
ich verstehe immer noch nicht, wieso wir es nicht erstmal einfach bei Lima-City lassen?! Dann brauchen wir lediglich eine Domain und einen Namen, den wir dort angeben können...

**Comment by kjoscha** [comment]
die zwei Datenbank-Downs gestern sind glaub ich eine absolute Ausnahmen. Die haben auch ansonsten einen sehr guten Support mit Live-Chat und so....

**Comment by andneuma** [comment]
sorry, ich schreibe grad ein wenig durcheinander. Also: dort ist 1&1, ich dachte auch, du würdest deren Webspace benutzen wollen. Wenn nicht - auch gut, ich hab da keine Aktien drin. Wenn du sagst, lima-city ist gut, dann ist das gut und wir lassen es da!

Zur Domain: Ich kläre grad übers Board, ob wir die Registrierung der Domain evtl. über einen der Hausvereine laufen lassen können. Kosten müssten dann als Spende zum Verein zurückfließen, logo. Ist bei 1-2€ / Monat aber ja auch nicht so der Hit.

**Comment by kjoscha** [comment]
> Also: dort ist 1&1, ich dachte auch, du würdest deren Webspace benutzen wollen

ah okay! Ich denke, wir könnten später immernoch wechseln, wenn Lima uns nicht ausreicht, oder?

> Zur Domain: Ich kläre grad übers Board, ob wir die Registrierung der Domain evtl. über einen der Hausvereine laufen lassen können. Kosten müssten dann als Spende zum Verein zurückfließen, logo. Ist bei 1-2€ / Monat aber ja auch nicht so der Hit.

Supi, danke!

**Comment by andneuma** [comment]
Jo, Vorschlag für Projekt mit bundesweitem Fokus

* http://refugee-ressources.org

=> subdomain zB germany.refugee-ressources.org, wenn man das sogar im europäischen Kontext denkt. Dann wirds aber richtig groß...

**Comment by magdalena19** [comment]
Also ich finde es garnicht unedingt so cool, refugee im Projektnamen zu tragen. Auch wenn diese natürlich unsere Grundmotivation darstellen und zumindest zu Beginn unsere "Zielgruppe" sind, möchte ich diese implizierte Kategorisierung am liebsten vermeiden. Weißt du was ich meine? Ich mache mir auch mal Gedanken...

**Comment by andneuma** [comment]
Jo, versteh ich. Is halt grad von der Ausrichtung schon sehr darauf zugeschnitten. Versteh aber dennoch den Punkt

Btw. immernoch keine news von Suse wegen Hosting, ich hak nochmal nach

**Comment by andneuma** [comment]
Domains kosten nicht die Welt. Wenn wir das selber machen gibts bei 1&1 grad Angebote für 1€ für ne .de-Domain.

**Comment by kjoscha** [comment]
nehmen wir nun einfach welcomemap und welcome-map?
